src/pages/utils.tsx

```diff
@@ -112,3 +112,15 @@ export function getTableFormattedTimestamp(timestamp?: string): string {
 
   return timestamp_display.local_formatted_short;
 }
+
+// example output: 02/09/2023, 01:38:22 PM
+export function prettifyTimestamp(timestamp: number) {
```

Comment:
I missed it in the last review. But can we reuse `<TimestampValue .../>`? We should keep the timestamps the same format across Explorer.